torch_xla/csrc/lowering_context.cpp

```diff
@@ -169,6 +169,33 @@ xla::StatusOr<xla::XlaComputation> LoweringContext::BuildXla() {
   return xla;
 }
 
+xla::StatusOr<xla::XlaComputation> LoweringContext::BuildXlaWithCheck() {
+  xla::StatusOr<xla::XlaComputation> xla;
+  if (!root_tuple_.empty() & (root_tuple_.size() > 1)) {
+    xla::XlaOp root = xla::Tuple(builder(), root_tuple_);
+    xla = builder()->Build(root);
+  } else if (!root_tuple_.empty() & (root_tuple_.size() == 1)) {
```

Comment:
we need to skip tuple for cond/body computation creation due to match the xla::While format check